showcase/app/components/mock/app/sidebar/app-side-nav.gts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+// HDS components
+import {
+  HdsAppSideNav,
+  HdsAppSideNavList,
+} from '@hashicorp/design-system-components/components';
+
+// types
+import type { HdsAppSideNavSignature } from '@hashicorp/design-system-components/components/hds/app-side-nav/index';
```

Comment:
[thought] My bad, I made an error when I created this. Looking at the AppHeader, the name used is `MockAppHeaderAppHeader` so maybe we should be consistent and have `MockAppSidebarAppSideNav`?

Reply:
[I know it's super confusing, and I regret my initial choice of having subfolders]

Reply:
All good, I think I got myself confused too with the naming. I'll update to `MockAppSidebarAppSideNav` and `MockAppSidebarSideNav`.